### PR TITLE
Support Equations

### PR DESCRIPTION
**Pull request by sHermanGriffiths**
Add support for inline equations and equation blocks.

### CODE REVIEW

File: equation block.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (johndgiese) [inline equation block.md:1]: Is this file being used?

File: equation inline.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (johndgiese) [inline equation inline.md:1]: Is this file being used?

File: n2y/converter.py [hunk 3]
```diff
@@ -133,8 +135,15 @@ def get_children(self):
                 previous_child_type = child['type']
 
 
+class Equation():
```
Comment (johndgiese) [inline n2y/converter.py:138]: Does `Equation` or `EquationBlock` need to be added to the readme?

File: tests/test_converter.py [hunk 4]
```diff
@@ -420,19 +421,48 @@ def test_strikeout_word():
     assert newline_lf(markdown_output) == expected_markdown
 
 
+def test_equation_inline():
```
Comment (johndgiese) [inline tests/test_converter.py:424]: Nice job adding some tests!

**Review by johndgiese — COMMENTED** [review]
@sHermanGriffiths at first glance this all looks great! Please request a more detailed technical review from @xaker00 

Also note it appears that the 3.8 test build appears to be failing, as evidenced by the little red "x" in the Github PR interface.

File: equation inline.md (hunk 2 above)
Comment (sHermanGriffiths) [reply equation inline.md]: no, I've just deleted

File: n2y/converter.py (hunk 3 above)
Comment (sHermanGriffiths) [reply n2y/converter.py]: Just `EquationBlock`, I've deleted `Equation`

**Comment by johndgiese** [comment]
@sHermanGriffiths your changes look good to me. Please request a review from @xaker00

**Pull request merged**